kernel/packages/shared/comms/v1/brokerWorldInstanceConnection.ts

```diff
@@ -237,6 +237,10 @@ export class BrokerWorldInstanceConnection implements WorldInstanceConnection {
     this.connection.close()
   }
 
+  analyticsData() {
+    return
```

Comment:
Let's return an empty object here to avoid issues if statistics happen to be sent from local cli connection

Reply:
I was thinking undefined could be part of the contract. I guess I could change the type to object | undefined

Reply:
Alright! 👍

Reply:
But wouldn't it result in a null reference error here https://github.com/decentraland/explorer/pull/747/files#diff-aa534620ff0363a557316b7564a11adeR676 ?

Reply:
You are right. Just fixed it!

Reply:
Amazing! 🌟